### PR TITLE
[openwrt] add Makefile

### PR DESCRIPTION
**Pull request by IHIDCHAOS**
Add Makefile for OpenWrt. 
tested with OpenWrt 19.07.

### CODE REVIEW

File: openwrt/ot-commissioner/Makefile [hunk 1]
```diff
@@ -0,0 +1,62 @@
+include $(TOPDIR)/rules.mk
```
Comment (wgtdkp) [inline openwrt/ot-commissioner/Makefile:1]: license header is missing.

File: .gitignore [hunk 2]
```diff
@@ -41,7 +41,7 @@ config.log
 config.status
 configure
 libtool
-Makefile
+#Makefile
```
Comment (wgtdkp) [inline .gitignore:44]: nit: it's ok to simply remove this.

**Review by wgtdkp — APPROVED** [review]
Thanks! LGTM 👍

File: .gitignore (hunk 2 above)
Comment (jwhui) [reply .gitignore:44]: Agree with removing this line.

**Review by jwhui — APPROVED** [review]

**Pull request merged**